receiver/hostmetricsreceiver/README.md

```diff
@@ -188,6 +190,9 @@ Currently, the hostmetrics receiver does not set any Resource attributes on the
 ```
 export OTEL_RESOURCE_ATTRIBUTES="service.name=<the name of your service>,service.namespace=<the namespace of your service>,service.instance.id=<uuid of the instance>"
 ```
+## Entity Events
+
+The hostmetrics receiver can send host entity event as log. See the [Entites Data Model](https://github.com/open-telemetry/oteps/blob/main/text/entities/0256-entities-data-model.md#entity-events) spec for more details. By default, hostmetrics receiver sends periodic EntityState event every 5 minutes. You can change that by setting `metadata_collection_interval`.
```

Comment:
Please mention that this is experimental log-based entities that eventually will be replaced by the result of the OTEP